src/components/SidebarLinks/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { Sword, MaskSad, Balloon, Ghost, HouseLine } from 'phosphor-react';
+import { NavLink } from 'react-router-dom';
+import { SidebarChildrenLinks } from '../SidebarChildrenLinks';
+import { Text, Link, Flex } from '@chakra-ui/react';
+export const SidebarLinks = () => {
+  return (
+    <>
+      <SidebarChildrenLinks>
+        <Link
```

Comment:
Bro, you can create a component about it, because you're repeating the same props in every part of this file

Reply:
You can check if have this correct?